common/model-views.cpp

```diff
@@ -3698,23 +3699,29 @@ namespace rs2
 
             //glTexParameterfv(GL_TEXTURE_2D, GL_TEXTURE_BORDER_COLOR, tex_border_color);
 
-            glBegin(GL_POINTS);
+            glBegin(GL_QUADS);
 
             auto vertices = last_points.get_vertices();
             auto tex_coords = last_points.get_texture_coordinates();
 
-            for (int i = 0; i < last_points.size(); i++)
-            {
-                if (vertices[i].z)
-                {
-                    glVertex3fv(vertices[i]);
-                    glTexCoord2fv(tex_coords[i]);
-                }
+#define THRESH 0.5f
```

Comment:
Please replace preprocessors with compiled code

Reply:
I left it as a preprocessor to indicate that the value isn't final.
I'm not happy with the value .5f, but I have yet to find a single hardcoded value that works well.
It's possible a more complex heuristic is necessary